### PR TITLE
WQP-1624 -- Update to use uuid for request identifier

### PR DESCRIPTION
**Pull request by orosz-usgs**
Before making a pull request
----------------------------

- [ ] Make sure all tests run
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: src/main/java/gov/usgs/wma/wqp/dao/LogDao.java [hunk 1]
```diff
@@ -98,6 +101,7 @@ private void log() {
 	private void log(ObjectNode json) {
 		JsonNode field;
 		if (json != null) {
+			json.put(ID, UUID.randomUUID().toString());
 			field = json.get(ID);
 			String id = field == null || field.isNull() ? null : field.asText();
```
Comment (ee-usgs) [inline src/main/java/gov/usgs/wma/wqp/dao/LogDao.java:106]: I don't think these two lines are needed:  Just keep your UUID and reuse its value on line 109 for 'id'.
Also, on line 108, its really hard to read the logic there w/o parenthesis.  I find it more readable like this:
`String stage = (field == null || field.isNull()) ? null : field.asText();`

**Review by ee-usgs — APPROVED** [review]
I marked one small change.  I'm not sure why we need a UUID, since the entire request is on one 'row' now.

**Review by ee-usgs — CHANGES_REQUESTED** [review]
Also, after a chat with Jim, it sounds like we can just remove the recordCount (and its AtomicInteger counter) completely.  Since its not unique, it is only misleading and someone hunting the logs would be better off using the timestamps

**Comment by orosz-usgs** [comment]
The id is passed up to the Log Service, up to the Controller by the
Log service.  Neither the log Service or the controllers were changed.
I think we should continue to return an unique id for each request,
as before. Do not know what assumptions the code is making.

**Pull request merged**